share/completions/brew.fish

```diff
@@ -1,15 +1,56 @@
+function __fish_brew_get_cmd
+  for c in (commandline -opc)
+    if [ (echo $c | string sub -l 1) != "-" ]
+      echo $c
```

Comment:
> You'll probably want to break here to get just the first non-option.
I've replaced all `(commandline -opc)` in this file with this function so I guess it's necessary to have it like this.